.github/workflows/build-verification.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           distribution: 'adopt'
       - name: Set up Gradle
         uses: gradle/gradle-build-action@v3
+        with:
+          develocity-access-key: ${{ secrets.DV_SOLUTIONS_ACCESS_KEY }}
       - name: Build with Gradle
         run: ./gradlew build -i
```

Comment:
This is not a change you introduced, but I'm wondering if logging at `info` level is necessary

Reply:
I think it can be removed.